kie-remote/kie-remote-jaxb/src/main/java/org/kie/services/client/serialization/jaxb/impl/deploy/JaxbDeploymentJobResult.java

```diff
@@ -27,7 +28,7 @@
 @XmlRootElement(name="deployment-job-result")
 @XmlAccessorType(XmlAccessType.FIELD)
 @JsonIgnoreProperties({"jobId"})
-public class JaxbDeploymentJobResult {
+public class JaxbDeploymentJobResult implements Serializable {
```

Comment:
because it is variable/data for executor and thus needs to be serialized and stored as part of the job to be loaded on execution time - async way